precode.go

```diff
@@ -55,7 +71,17 @@ func main() {
 	var wg sync.WaitGroup
 
 	// 4. Собираем числа из каналов outs
-	// ...
+	for i, out := range outs {
+		wg.Add(1)
+		go func(i int, out <-chan int64,) {
```

Comment:
Лучше менять название переменной, чтобы точно понимать какая из них берется
```suggestion
		go func(j int, out <-chan int64,) {
```